cypress/e2e/awx/resources/executionEnvironments.cy.ts

```diff
@@ -265,15 +271,15 @@ describe.skip('Execution Environments: Edit and Bulk delete', () => {
     cy.verifyPageTitle(executionEnvironment.name + '-edited');
     cy.hasDetail('Name', executionEnvironment.name + '-edited');
     cy.hasDetail('Image', 'executionenvimage');
-    cy.hasDetail('Organization', 'Global Organization');
+    cy.hasDetail('Organization', organization.name);
   });
 
   it('can bulk delete multiple EEs from the list view and assert deletion', function () {
```

Comment:
@acruzgon do we still need to use function here? Since we are no longer `this`. Perhaps we could update just to be consistent everywhere that is not longer necessary.

Reply:
@nixocio agree! it's not needed anymore so i'll remove it.